Ecommerce/EcommerceWeb/wwwroot/js/order.js

```
@@ -1,7 +1,7 @@
-﻿var dataTable;
+﻿let dataTable;
 
 $(document).ready(function () {
-    var url = window.location.search;
+    let url = window.location.search;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'url' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)